_posts/--- layout: slide title: "Welcome to our second slide!" --- Y0000-01-02-RickGiannini.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![I-AM-NOT-PREPARED-FOR-THIS](https://user-images.githubusercontent.com/79111260/107996494-874e9980-6fae-11eb-8b14-de0edc815154.png)